highfidelity/hifi
# Adopt path to certain Tablet dialogs to new loading subsystem and pat…

Adopt path to certain Tablet dialogs to new loading subsystem and path resolver

https://highfidelity.fogbugz.com/f/cases/10854/The-Running-Scripts-dialog-and-Asset-Browser-windows-won-t-open-in-HMD

Test steps:
- Open tablet -> Menu -> Edit
- Open Running scripts
- Open Asset Browser
- Make sure dialogs are opened
